src/components/Stories/index.js

```diff
@@ -66,19 +65,24 @@ function Stories({
       {page === 1 && (
         <FeaturedStoryCard {...featuredStoryProps} variant={variant} />
       )}
-      {isLoading && <Loading />}
-      <List
-        items={items}
-        key={category}
-        variant={variant}
-        ctaText={featuredStoryProps.ctaText}
-      />
-      <Pagination
-        count={Math.ceil((pagination?.offsetPagination?.total ?? 0) / 9)}
-        onChangePage={handleClickPage}
-        page={page}
-        pageSize={9}
-      />
+      {isLoading ? (
+        <Loading />
+      ) : (
+        <>
+          <List
```

Comment:
Can we show Loading and keep the current stories? Right now clicking 2nd page clears all the current stories.